asserts/repair.go

```diff
@@ -55,6 +55,11 @@ func (r *Repair) RepairID() int {
 	return r.id
 }
 
+// Sequence implements SequenceMember, it returns the same as RepairID.
+func (r Repair) Sequence() int {
```

Comment:
Is there a reason it's not pointer type (r *Repair)?

Reply:
no, it's actually wrong, thanks for spotting this